packages/commerce-sdk-react/src/auth/storage/cookie.ts

```diff
@@ -33,8 +33,11 @@ export class CookieStorage extends BaseStorage {
         const suffixedKey = this.getSuffixedKey(key)
         return Cookies.get(suffixedKey) || ''
     }
-    delete(key: string) {
+    delete(key: string, options?: Cookies.CookieAttributes) {
         const suffixedKey = this.getSuffixedKey(key)
-        Cookies.remove(suffixedKey)
+        Cookies.remove(suffixedKey, {
+            ...getDefaultCookieAttributes(),
+            ...options
+        })
```

Comment:
Can you please update the CHANGLOG.md file. I think it's safe for us to call this out as a bug since we were setting cookies with those default attributes and then removing them without them.